sdcm/nemesis.py

```diff
@@ -1186,9 +1186,12 @@ def _add_and_init_new_cluster_node(self, old_node_ip=None, host_id=None,
         self.set_current_running_nemesis(node=new_node)  # prevent to run nemesis on new node when running in parallel
 
         # since we need this logic before starting a node, and in `use_preinstalled_scylla: false` case
-        # scylla is not yet installed, we should check the target node for version,
+        # scylla is not yet installed or node was terminated, we should the alive node without nemesis for version,
         # it should be up and with scylla executable available
-        if self.target_node.is_replacement_by_host_id_supported:
+        node_without_nemesis = [node for node in self.cluster.nodes if not node.running_nemesis]
```

Comment:
Not critical, but it could be simplified to the following:

```
verification_node = random.choice([
    node for node in self.cluster.nodes
    if node not in (new_node, self.target_node) and not node.running_nemesis
])
```

Reply:
Don't like to have multiline list comprehension. Fixed